### PR TITLE
FVCOM on ARM

### PR DESCRIPTION
**Pull request by Yikun**

### CODE REVIEW

File: FVCOM.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+|  Kernel | 4.14.0-115  |  Included in the OS image.        |
+## Configuring the Compilation Environment
+### Installing GNU 9.3
+
+    yum install -y centos-release-scl
+    yum install -y devtoolset-9-gcc
+    yum install -y devtoolset-9-gcc-c++
+    yum install -y devtoolset-9-binutils
+    scl enable devtoolset-9 bash
+    echo "souce /opt/rh/devtoolset-9/enable" >> /etc/profile
```
Comment (martin-g) [inline FVCOM.md]: s/souce/source/

File: FVCOM.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+### Installing Open MPI
```
Comment (martin-g) [inline FVCOM.md:37]: Add Open MPI to the table with software dependencies

File: FVCOM.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+    make -j 16
+    make install
+2. Configure environment variables:
+
+    export PATH=/path/to/netcdf/bin:$PATH
+    export LD_LIBRARY_PATH=/path/to/netcdf/lib:$LD_LIBRARY_PATH
+### Installing NETCDF-FORTRAN
+1. Run the following commands to install NETCDF-FORTRAN:
+
+    wget https://github.com/Unidata/netcdf-fortran/archive/refs/tags/v4.5.2.tar.gz
```
Comment (martin-g) [inline FVCOM.md]: needs `-o netcdf-fortran-4.5.2.tar.gz`

File: FVCOM.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+    cd julian
+    make -j
+    make install
+
+8. Run the following command to modify some source code:
+
+    cd /path/to/FVCOM/FVCOM4.1/FVCOM_source
+
+    vim mod_newinp.F
+   Line 343:
```
Comment (martin-g) [inline FVCOM.md]: Would it be better if a .patch file is created and put in this Git repo next to FVCOM.md file ?

**Review by martin-g — COMMENTED** [review]
Same improvements could be made as suggested in the other PRs

File: FVCOM.md (hunk 1 above)
Comment (qinluyao761) [reply FVCOM.md]: done

File: FVCOM.md (hunk 3 above)
Comment (qinluyao761) [reply FVCOM.md]: done

File: FVCOM.md (hunk 4 above)
Comment (qinluyao761) [reply FVCOM.md]: Good idea. We're preparing a patch.

**Pull request merged**